src/reporter/listing.ts

```diff
@@ -0,0 +1,30 @@
+import { DiagnosticSource, DiagnosticKind } from '../diagnostic/diagnostic';
+import { ColorCodes } from './common';
+
+export function diagnosticColor1(kind: DiagnosticKind): string {
+  switch (kind) {
+    case DiagnosticKind.Linter:
+      return ColorCodes.Green + ' lint ' + ColorCodes.Reset;
+    case DiagnosticKind.Error:
+      return ColorCodes.Red + ' error ' + ColorCodes.Reset;
+    default:
+      return ColorCodes.Yellow + ' warning ' + ColorCodes.Reset;
+  }
+}
+
+export function listing(
+  diagnosticSource: DiagnosticSource,
+  kind: DiagnosticKind,
+  message: string,
+  start: number,
+  end: number,
+  source: string
+) {
+  const lnSpacing = '_'.repeat(90);
+  process.stdout.write(`${lnSpacing}\n\n`);
+  process.stdout.write(
+    `${ColorCodes.Cyan + start}:${end + ColorCodes.Reset}${diagnosticColor1(kind)}${
+      ColorCodes.White + message + ColorCodes.Reset
+    }\n`
```

Comment:
@aladdin-add  To be equal with ESLint need to count the amount of errors, warning and lint failues so we can print 
'99 errors. 2 warnings. 1 lint issue (3 auto-fixed)'.